packages/core/src/components/AvatarGroup/AvatarGroupCounter.module.scss

```diff
@@ -51,6 +51,12 @@
     color: var(--primary-text-color);
     background-color: var(--ui-background-color);
   }
+
+  .disabled,
+  &.disabled {
+    background-color: var(--disabled-background-color);
+    color: var(--disabled-text-color);
+  }
```

Comment:
both selectors are declared on purpose?

Reply:
Yes, because one is a child and one is adjacent and I need to target both :(